lib/index.js

```diff
@@ -5,6 +5,35 @@ $(document).ready(() => {
   foodsRequests.getFoods()
   $('#submit-food').on('click', event => {
     event.preventDefault()
-    foodsRequests.addNewFood()
+    newFoodSequence()
   })
 })
+
+const newFoodSequence = () => {
+  var nameInput = $('#food-name').val()
+  var caloriesInput = $('#food-calories').val()
+  clearAlerts()
+  if (nameInput != '' && caloriesInput != '') {
+    foodsRequests.addNewFood()
+  } else if (nameInput == '' && caloriesInput == '') {
+    validateNameAlert()
+    validateCaloriesAlert()
+  } else if (caloriesInput == '') {
+    validateCaloriesAlert()
+  } else {
+    validateNameAlert()
+  }
+}
+
+const validateNameAlert = () => {
```

Comment:
I'm thinking we should refactor these alerts into a single method that takes a selector as a parameter. We could then pass the selector into the first parens.

`
const foodAlert = (selector) => {
 $(selector).css('display', 'block')
}
`